_docs/vision.md

```diff
@@ -59,5 +59,4 @@ people improve the user experience, raising it to new levels.
 
 Be part of the community and help to improve Babelfish even more.
```

Comment:
Be part of the community, and help us improve Babelfish.

Reply:
Done in 70381500788c2240882fcbcde9c3edb6ec461886.